web/modules/custom/os2forms_email/os2forms_email.info.yml

```
@@ -0,0 +1,7 @@
+name: 'OS2Forms email'
+type: module
+description: 'Overrides email handler adding extra checks and functionality'
+package: 'OS2Forms'
+core_version_requirement: ^9
```

Work is in progress to upgrade OS2Forms to Drupal 10:

```suggestion
core_version_requirement: ^9 || ^10
```